docs/CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+  * **places**: Add `population_rank` to `locality`, `country`, and other kinds. A value from 0 (no population) up to 18 (over 1 billion people).
+  * **traffic_flow**: Add `road_kind_detail` to enable data-driven client-side traffic line offseting, thanks @zaczkows! (Issue [#1829](https://github.com/tilezen/vector-datasource/issues/1829))
+  * **all layers**: Add `collision_rank` property for label collisions for **pois** layer, and other layers with features with `name` properties, or `ref`, `shield_text`, `bicycle_shield_text`, `bus_shield_text`, or `walking_shield_text`. We now recommend colliding labels client side first with `min_zoom`, and then to tie break with the new `collision_rank` values. (Issue [#988](https://github.com/tilezen/vector-datasource/issues/988))
+
+  #### BUG FIXES
+
+  * **landuse**: Fix v1.6 regression where `aerodrome` polygons sorted above `runway` and `taxiway` polygons in error, thanks @bcamper! (Issue [#1814](https://github.com/tilezen/vector-datasource/issues/1814))
+  * **landuse**: Add additional filter for newer OSM `area:aeroway` tagging of `runway`, `taxiway`, and `apron` polygons to restore features from prior year's tile builds. (Issue [#1814](https://github.com/tilezen/vector-datasource/issues/1814))
+  * **landuse**: Restore national forests `forests` to zoom 7 (Issue [#1801](https://github.com/tilezen/vector-datasource/issues/1801))
+  * **landuse**: "park" in Marin shouldn't be `national_park` (instead `common`) (Issue [#1082](https://github.com/tilezen/vector-datasource/issues/1082))
+  * **landuse**: Henry Coe State Park is incorrectly marked as kind `national_park` (Issue [#1728](https://github.com/tilezen/vector-datasource/issues/1728))
+  * **landuse**: Features of kind `mud` now sorts above `water` and below `wetland` (Issue [#1753](https://github.com/tilezen/vector-datasource/issues/1753))
+  * **landuse**: Demoted many `nature_reserves` from zoom 10 (Issue [#1749](https://github.com/tilezen/vector-datasource/issues/1749))
+  * **landuse**: Landcover related landuse should all show at zoom 9 (Issue [#1813](https://github.com/tilezen/vector-datasource/issues/1813))
+  * **landuse**: Demote some early landcover down a few zooms (Issue [#1794](https://github.com/tilezen/vector-datasource/issues/1794))
+  * **landuse**: BLM managed `protected_areas` are now filtered separately.
+  * **landuse**: Pull Yellowstone (and other such huge national parks) up to z5. Don't…  …
+  * **landuse**: Add special cases for BLM-managed areas and things in 'strict' or 'sc…  …
+  * **landuse**: Harmonise kind assignment for national parks, forests, etc... between…  …
+  * **landuse**: Add `protection_title` based filtering for national parks.
+  * **landuse**: Promote some landcover-ish landuses up a few zooms to zoom 9, like `farm` and `farmland`, `orchard`.
+  * **landuse**: Push `forest` down a little further.
```

Comment:
The seven lines above this look like single commit messages, and aren't very specific. If we can't link them back to an issue, suggest we reword them to explain what they actually do. Otherwise, "a little further" might just be confusing to anyone reading this?

Reply:
Yikes, that was a mess. Fixed in 9de0449363c65f798977b40a85e16070d501b3ee.